src/components/Card.js

```diff
@@ -74,31 +49,108 @@ const Card = ({ content, dataPolicy, protocols, sampleUnits, histogram }) => {
       <Box pt={1}>
         <Chart
           chartType={type}
-          chartContent={body}
+          chartContent={pieContent}
           chartLegend={legend}
-          chartPolicy={dataPolicy}
+          privatePolicy={privatePolicy}
+          privateLabel={privateLabel}
           histogram={histogram}
         />
       </Box>
     );
 
-  const contentAvailable = content ? (
-    <Paper className={classes.cardWrapper}>
-      <Box display="flex" borderBottom={1}>
-        <Box flexGrow={1}>
-          <Box>
-            <Typography variant="h4">{title}</Typography>
+  return <>{content}</>;
+};
+
+const Card = ({
```

Comment:
It's now clear that this "Card" serves a specialized purpose. You should prepend the name of this React Class as well as the filename so that the purpose of this Card becomes more clear to a developer